llama-index-core/llama_index/core/workflow/workflow.py

```diff
@@ -69,7 +69,7 @@ async def _task(
                     new_ev = await step(ev)
 
                     if self._verbose:
-                        print(f"Step {name} produced event {new_ev}")
+                        print(f"Step {name} produced event {type(new_ev).__name__}")
```

Comment:
Trying to prevent casting the event as a string (which will exhaust generators, and also just fill your terminal with nonsense)

Reply:
Indeed, I planned to add a `__repr__` method to the Event class, we can do that later